### ISSUE TITLE
question about the setting of custom diffusion and dreambooth

### ISSUE THREAD

**Issue by x2x5** [Rbrq03/ClassDiffusion#14]
Hi, I have been reading about the Custom Diffusion method, which is a baseline for your work. In the original paper, the authors mention using a set of regularization images (for instance, ~200 images of a base class like "cat") that are retrieved using CLIP-Retrieval.

I've learned that the official LAION clip-retrieval service has been discontinued, which makes it challenging to acquire these class-specific images. This led me to a question about your own implementation: did you incorporate the ~200 regularization images per class when you were setting up your experiments with Custom Diffusion?

Furthermore, I was wondering if the primary difference between your method and Custom Diffusion (and DreamBooth) lies in the loss function and keep other parts same.
For example, original DreamBooth finetunes the whole unet. Does the baseline DreamBooth here finetunes the whole unet or the key-value part in the unet?

Of course, these question don't affect from the excellence of your work. As I am just beginning to study image personalization, I would be really grateful for any further details you could share about your comparative setup for this task.

Thank you very much for your time and consideration~

**Comment by Rbrq03** [comment]
Hi, thank you for your interest!

For the first question, no. I am confused by this question also, so for all baseline and our implementation, we don't it.

For the second question. We are the same as the CustomDiffusion except for the loss function part.

I hope it can adress your question, and feel free to reach if you have further question!
